config/vufind/SierraRest.ini

```diff
@@ -268,3 +268,18 @@ title_hold_excluded_item_codes = "e"
 ;display_orders = false
 ; Whether to check holdings records in search results (default is true):
 ;check_holdings_in_results = true
+
+; This section can be configured to allow display of held and checked out items from
+; INN-Reach systems
+[InnReach]
+; In order to get title information from INN-Reach we need to connect to Sierra DNA
+dna_url = [DNA address]
```

Comment:
Why not make the configuration a connection string instead of the separate settings that you need to parse together? That would enable any connection string parameters like `sslmode` without additional effort.